File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/attributes/cs/workwithattributes7.cs
// <snippetworkwithattributes7>


 // Create a request.
 InsertOptionValueRequest insertOptionValueRequest =
     new InsertOptionValueRequest
 {
     AttributeLogicalName = "new_picklist",
     EntityLogicalName = Contact.EntityLogicalName,
     Label = new Label("New Picklist Label", _languageCode)
 };

 // Execute the request.
 int insertOptionValue = ((InsertOptionValueResponse)_serviceProxy.Execute(
     insertOptionValueRequest)).NewOptionValue;

 Console.WriteLine("Created {0} with the value of {1}.",
     insertOptionValueRequest.Label.LocalizedLabels[0].Label,
     insertOptionValue);

// </snippetworkwithattributes7>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/attributesvb/vb/workwithattributes7.vb
' <snippetworkwithattributes7>


 ' Create a request.
 Dim insertOptionValueRequest As InsertOptionValueRequest = New InsertOptionValueRequest With {
  .AttributeLogicalName = "new_picklist",
  .EntityLogicalName = Contact.EntityLogicalName,
  .Label = New Label("New Picklist Label", _languageCode)}

 ' Execute the request.
 Dim insertOptionValue As Integer = (CType(_serviceProxy.Execute(insertOptionValueRequest), InsertOptionValueResponse)).NewOptionValue

 Console.WriteLine("Created {0} with the value of {1}.", insertOptionValueRequest.Label.LocalizedLabels(0).Label, insertOptionValue)

' </snippetworkwithattributes7>